### DISCUSSION TITLE
v3.5.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#318] [Announcements]
### New Features

- Support long-press to speeding the playback

### Bug Fixes

- Videos can not be played (YouTube)
- Crash when searching specific words (YouTube)
- Unable to export subscriptions sometimes (YouTube)
- Download post-processing failed sometimes(BiliBili)
- Incorrect playlist order under certain conditions

### Performance

- Disable auto-fallback to avoid long buffer time

<hr/>
If you're enjoying PipePipe, please help share it with your friends. Every bit of support means a lot to us!
<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.5.0'>v3.5.0</a>.</em>

**Comment by jross200** [comment]
Thank you so much; just tossed a donation your way via kofi.

**Reply by InfinityLoop1308** [reply]
Thank you for your support!
